### PR TITLE
Remove hhvm.hhbc and pid file from the package

### LINKED ISSUE #83: Remove hhvm.hhbc from Pre-Built Packages
hhvm.hhbc should be removed from prebuilt-packages as it often creates file ownership and permissions issues.
[Source][1]
[1]:https://github.com/facebook/hhvm/issues/4089#issuecomment-64842563

### LINKED ISSUE #109: eager commits to packaging repo
It seems odd that this directory and contents are even part of the packaging repository
  https://github.com/hhvm/packaging/tree/master/hhvm/deb/skeleton/var/run/hhvm

Among other things, a system wide purge of any files hhvm.hhbc will result in git later squawking about this file being missing from its purview.

### PR DESCRIPTION
**Pull request by bauerj**
Closes #109
Closes #83

### CODE REVIEW

**Comment by bauerj** [comment]
It would of course be easier to just add the empty directory to the `skeleton` directory. But that would mean we would need to create a `.gitignore` or something so git would add it, which would in turn be quite confusing to have inside a package.

File: hhvm/deb/skeleton/DEBIAN/postinst [hunk 1]
```diff
@@ -38,6 +38,12 @@ echo "*    --install /usr/bin/php php /usr/bin/hhvm 60"
 
 echo "********************************************************************"
 
+if [ ! -d "/var/run/hhvm" ]; then
+    mkdir -p /var/run/hhvm
+    chmod 644 /var/run/hhvm
+    chown www-data:www-data /var/run/hhvm
```
Comment (glensc) [inline hhvm/deb/skeleton/DEBIAN/postinst]: 644 is not good mode for directory! you did not seem to actually test your changes!

and why not use install(1) to make whole change in one command?

```
install -d -m 755 -o www-data -g www-data /var/run/hhvm
```

**Comment by bauerj** [comment]
Thanks @glensc.

I did of course test the updated postinst but it seemed to work. You're of course right that leaving directories without an executable bit is a bad idea :grin: 

I updated my pull request with the line you gave me, thank you!

**Comment by jwatzman** [comment]
Looks good to me, thanks @bauerj and @glensc!

**Pull request merged**